src/auth/auth.guard.ts

```diff
@@ -0,0 +1,25 @@
+import {
```

Comment:
`AuthGuard` implementation. Only allows requests using this guard to continue if the `x-userid` header is present and not an empty string (all real-world headers will be strings)